pkg/server/apiHandler.go

```diff
@@ -350,15 +348,14 @@ func (h *apiHandler) logs(w http.ResponseWriter, r *http.Request) {
 			log.Printf("failed to marshal protobuf message into json: %v", err)
 			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 		}
-		buf.Write(entryBytes)
+		err = h.manager.WriteLogs(*u, entryBytes)
+		if err != nil {
+			log.Printf("Failed to write logbundle message: %v", err)
```

Comment:
Just small remark: there is not logbundle here (logbundle->fulllogentry).

Reply:
Good point. Fixed. Anything else?